apps/metadata-editor/src/app/my-org-users/my-org-users.component.ts

```diff
@@ -1,48 +1,27 @@
-import { Component, OnDestroy } from '@angular/core'
-import { RecordsListComponent } from '../records/records-list.component'
+import { Component } from '@angular/core'
 import { UiInputsModule } from '@geonetwork-ui/ui/inputs'
 import { TranslateModule } from '@ngx-translate/core'
 import { CommonModule } from '@angular/common'
 import { MyOrgService } from '@geonetwork-ui/feature/catalog'
-import { SearchFacade } from '@geonetwork-ui/feature/search'
-import { UserModel } from '@geonetwork-ui/common/domain/model/user/user.model'
+import { UsersListComponent } from './users-list.component'
+import { UiElementsModule } from '@geonetwork-ui/ui/elements'
+import { tap } from 'rxjs'
 
 @Component({
   selector: 'md-editor-my-org-users',
   templateUrl: './my-org-users.component.html',
   styleUrls: ['./my-org-users.component.css'],
   standalone: true,
   imports: [
-    RecordsListComponent,
     UiInputsModule,
     TranslateModule,
     CommonModule,
+    UsersListComponent,
+    UiElementsModule,
   ],
 })
-export class MyOrgUsersComponent implements OnDestroy {
-  orgData: {
-    orgName: string
-    logoUrl: string
-    recordCount: number
-    userCount: number
-    userList: UserModel[]
-  }
+export class MyOrgUsersComponent {
```

Comment:
This component is already not doing much in terms of logic, would it cost much to make it dumb?